rest/data/commit_queue.go

```diff
@@ -280,6 +283,103 @@ func (pc *DBCommitQueueConnector) IsAuthorizedToPatchAndMerge(ctx context.Contex
 	return hasPermission, nil
 }
 
+// EnqueuePRToCommitQueue enqueues an existing pull request to the commit queue.
```

Comment:
The phrasing on the doc comment might be a little vague, since it's unclear it means to "enqueue a pull request". Can we phrase this as something more Evergreen-ish, like "enqueues a commit queue item to test and merge a PR"?

Reply:
Done